Issue resolving dependencies with stylelint 14.x

I received this error when installing with `stylelint: 14.x` It worked fine when running so i suppose all that needs to happen is to also support v14 in the peerDependencies section. 
```
 Could not resolve dependency:
npm ERR! peer stylelint@"^13.0.0" from lint-to-the-future-stylelint@0.1.1
npm ERR! node_modules/lint-to-the-future-stylelint
npm ERR!   dev lint-to-the-future-stylelint@"*" from the root project
```

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "release-it": "^14.4.1"
   },
   "peerDependencies": {
-    "stylelint": "^13.0.0"
+    "stylelint": "^13.0.0 || ^14.0.0"
   },
   "engines": {
     "node": "12.* || >= 14.*"

